frontend/src/routes/create-goals/+page.svelte

```diff
@@ -95,211 +80,379 @@
-                        <div class="absolute inset-0 bg-gray-500 opacity-75" />
-                      </div>
-                      <div
-                        class="inline-block align-bottom bg-white rounded-lg text-left overflow-hidden shadow-xl transform transition-all sm:align-middle sm:max-w-lg sm:w-full sm:my-8"
-                      >
-                        <div class="bg-white px-4 pt-5 pb-4 sm:p-6 sm:pb-4">
-                          <div class="sm:flex sm:items-start">
-                            <!-- Modal content goes here -->
-                            <div>
-                              <h3 class="text-lg leading-6 font-medium text-gray-900">
-                                Edit Smart Goal
-                              </h3>
-                              <div class="mt-2">
-                                <!-- Add your form fields here -->
-                              </div>
-                            </div>
-                          </div>
-                        </div>
-                        <div class="bg-gray-50 px-4 py-3 sm:px-6 sm:flex sm:flex-row-reverse">
-                          <button
-                            on:click={handleEdit}
-                            class="w-full inline-flex justify-center rounded-md border border-transparent shadow-sm px-4 py-2 bg-primary text-base font-medium text-white hover:bg-secondary focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-secondary sm:ml-3 sm:w-auto sm:text-sm"
-                          >
-                            Edit
-                          </button>
-                          <button
-                            on:click={closeModal}
-                            class="mt-3 w-full inline-flex justify-center rounded-md border border-gray-300 shadow-sm px-4 py-2 bg-white text-base font-medium text-gray-700 hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-gray-50 sm:mt-0 sm:w-auto sm:text-sm"
-                          >
-                            Close
-                          </button>
-                        </div>
-                      </div>
+    <div class="card w-full">
+      <figure>
+        <figcaption class="p-4 card-body flex flex-row items-center">
+          <h2 class="card-title mb-2">Date</h2>
+          <div class="flex-grow flex items-center relative">
+            <input
+              class="shadow appearance-none border rounded w-full py-2 px-3 leading-tight focus:outline-none focus:shadow-outline"
+              id="goal-date"
+              type="date"
+              bind:value={goal.date_for_achievement}
+              aria-describedby="date-description"
+            />
+            <label
+              class="cursor-pointer label flex items-center md:ml-2 mt-2 md:mt-0 w-full md:w-auto justify-end"
+            >
+              <input type="checkbox" class="toggle toggle-primary" checked />
```

Comment:
We don't need to be able to lock this

```suggestion
```

Reply:
I corrected all the instances like this on this page.